doc/release-notes.md

```diff
@@ -62,16 +62,18 @@ We now use a new block template construction algorithm documented in
   "unpaid actions" that will be accepted in a block for transactions paying less
   than the ZIP 317 fee. This defaults to 50.
 
-Removal of Priority Estimation
-------------------------------
+Removal of Priority and of Fee Estimation
+-----------------------------------------
 
-- Estimation of "priority" needed for a transaction to be included within a target
-  number of blocks, and the associated `estimatepriority` RPC call, have been
-  removed. The format for `fee_estimates.dat` has also changed to no longer save
-  these priority estimates. It will automatically be converted to the new format
-  which is not readable by prior versions of the software. The `-txconfirmtarget`
-  config option is now obsolete and has also been removed. It will cause a
-  warning if used.
+- The concept of transaction "priority" has been entirely removed. Also, the
+  `estimatepriority` and `estimatefee` RPC calls, which estimated the priority
+  and fee needed for a transaction to be included within a target number of blocks,
+  have been removed. The fee that should be paid under normal circumstances is
+  the ZIP 317 conventional fee; it is not possible to compute that without knowing
+  the number of logical actions in a transaction, which was not an input to
+  `estimatefee`. The `fee_estimates.dat` file is no longer used.
+- The `-txconfirmtarget` config option is now obsolete and has also been removed.
+  It will cause a warning if used.
```

Comment:
Now that the `version-5.5.0` release branch has been cut, the `release_notes.md` file can be emptied and just the new content can be added (I'm assuming that we're not intending to add this during the RC period, but if I'm wrong we can keep this as it is for now.)

Reply:
Right; I'll rebase this after the 5.5.0 release branch has been merged back to master.

Reply:
> Now that the `version-5.5.0` release branch has been cut, the `release_notes.md` file can be emptied

No it can't; the `release_notes.md` file remains a staging file until the final v5.5.0 release is cut (as its contents are reused for both subsequent RCs and the final release).

Reply:
The contents of this file only matters now on the 'version-' branch, the whole point is being able to keep merging to 'main' once the release stabilization branch is cut.